src/java.se/share/data/jdwp/jdwp.spec

```diff
@@ -2021,10 +2021,10 @@ JDWP "Java(tm) Debug Wire Protocol"
         (Reply "none"
         )
         (ErrorSet
-            (Error INVALID_THREAD "The thread is null, not a valid thread, the thread "
-                                  "is not alive, or the thread is a virtual thread "
-                                  "and the target VM does not support stopping it "
-                                  "with an asynchronous exception at this time.")
+            (Error INVALID_THREAD "The thread is null, not a valid thread, or the thread "
+                                  "is not alive.")
+            (Error NOT_IMPLEMENTED "The thread is a virtual thread and the target VM "
+                                  "VM does not support the operation.")
```

Comment:
The updated version repeats "VM".

Reply:
Oops. Already integrated. Will do a direct push to fix. Also, I think the JDI wording should be made to look similar.

Reply:
I also think it should be "command" instead of "operation" to be consistent with the text a little bit above this that reads:

        "The target VM may not support, or may only provide limited support, for "
        "this command when the thread is a virtual thread. It may, for example, "

Reply:
> I also think it should be "command" instead of "operation" to be consistent with the text a little bit above this that reads:

Okay, although the resulting "The thread is a virtual thread and the target VM does not support the command" should probably be expanded to make it clear that the command is not supported on virtual threads. This would avoid an interpreting it to mean that the error is returned when the VM doesn't support the command AND the thread ID is for a virtual thread.

Reply:
https://github.com/openjdk/loom/commit/5a2fbb7438e551ffbecedb5530d08bae5d79b74d

I tried to be consistent with the descriptive text above the error/exception section, thus "the command" for JDWP and "this operation" for JDI.